src/lightning/pytorch/callbacks/progress/rich_progress.py

```diff
@@ -207,16 +207,43 @@ class RichProgressBarTheme:
 
     """
 
-    description: Union[str, Style] = "white"
-    progress_bar: Union[str, Style] = "#6206E0"
-    progress_bar_finished: Union[str, Style] = "#6206E0"
-    progress_bar_pulse: Union[str, Style] = "#6206E0"
-    batch_progress: Union[str, Style] = "white"
-    time: Union[str, Style] = "grey54"
-    processing_speed: Union[str, Style] = "grey70"
-    metrics: Union[str, Style] = "white"
-    metrics_text_delimiter: str = " "
-    metrics_format: str = ".3f"
+    @staticmethod
+    def detect_color_system() -> str:
+        console = Console()
+        return console.color_system
```

Comment:
Couldn't we make this a proper function? 
Is this the recommended way how users should implement their themes with rich?